specification/gedcom-2-data-types.md

```diff
@@ -339,8 +339,8 @@ Special = Text
 ## File Path
 
 The file path data type describes where an digital file is located in a machine-readable way.
-Syntactically, the payload is a URI reference as defined by [RFC 3986](https://www.rfc-editor.org/info/rfc3986), or a valid URL string as defined by the [WHATWG URL specification](https://url.spec.whatwg.org/).
-That is, it can be an absolute or relative URL, optionally with a fragment string.
+Syntactically, the payload is a "valid URL string" as defined by the [WHATWG URL specification](https://url.spec.whatwg.org/).
+That is, it can be an absolute or relative URL, optionally with a fragment string, and can contain non-ASCII characters that are permitted in a valid URL string.
```

Comment:
```suggestion
That is, it can be an absolute or relative URL, optionally with a fragment string, and can contain non-ASCII characters that are permitted in a valid URL string; it can also contain characters represented as percent-encoded UTF-8 bytes.
```

If we mention percent encoding elsewhere, I think we should here too.

Reply:
A "valid URL string" can also contain percent-encoded bytes that are not percent-encoded UTF-8 bytes, so I wonder if we should use different wording?  https://url.spec.whatwg.org/#url-units says "The URL units are [URL code points](https://url.spec.whatwg.org/#url-code-points) and [percent-encoded bytes](https://url.spec.whatwg.org/#percent-encoded-byte)."

(i.e.: URL code points are Unicode characters, but percent-encoded bytes need not be)

Reply:
Updated text.